jmix-flowui/flowui-kit/src/main/java/io/jmix/flowui/kit/component/main/ListMenu.java

```diff
@@ -617,6 +692,73 @@ public MenuItem withShortcutCombination(@Nullable KeyCombination shortcutCombina
             return this;
         }
 
+        /**
+         * @return suffix component of the item
+         */
+        @Nullable
+        public Component getSuffixComponent() {
+            return suffixComponent;
+        }
+
+        /**
+         * Sets suffix component of the item.
+         *
+         * @param suffixComponent component to set
+         * @return current menu item instance
+         */
+        public MenuItem withSuffixComponent(@Nullable Component suffixComponent) {
+            this.suffixComponent = suffixComponent;
+            return this;
+        }
+
+        /**
+         * Sets suffix component of the item.
+         *
+         * @param suffixComponent component to set
+         */
+        public void setSuffixComponent(@Nullable Component suffixComponent) {
```

Comment:
Let's add `setter` methods for the other properties to make it consistent and more convenient, since it's hard to figure it out that there are `withXXX` methods.

Reply:
We decided not to implement additional setter methods now because common property setting logic (subscribtion to some property-change events) should be refactored first.